Documentation: Console Errors on DevDocs

The devdocs still work, but return a few console errors when viewing.

`TypeError: config.showFilters is not a function`
`TypeError: Cannot convert undefined or null to object`
`Uncaught (in promise) Error: Cannot find module './chart/example.md'`

**To Reproduce**

1. Open the developer console
2. Go to `http://:site/wp-admin/admin.php?page=wc-admin#/devdocs/`
3. See console errors

**Screenshots**
<img width="463" alt="screen shot 2018-12-03 at 11 46 41 am" src="https://user-images.githubusercontent.com/689165/49388107-55183080-f6f1-11e8-8b13-55585fb67c7f.png">

**Desktop (please complete the following information):**

* OS: Mac
* Browser Chrome
* Version 70.0.3538.110

--- a/packages/components/src/filters/example.md
+++ b/packages/components/src/filters/example.md
@@ -1,56 +1,91 @@
 ```jsx
 import { ReportFilters } from '@woocommerce/components';
+const { orderStatuses } = wcSettings;
 
 const path = '';
 const query = {};
 const filters = [
-	{ label: 'All Products', value: 'all' },
-	{ label: 'Top Products by Items Sold', value: 'top_items' },
-	{ label: 'Top Products by Gross Sales', value: 'top_sales' },
+	{
+		label: 'Show',
+		staticParams: [ 'chart' ],
+		param: 'filter',
+		showFilters: () => true,
+		filters: [
+			{ label: 'All Orders', value: 'all' },
+			{ label: 'Advanced Filters', value: 'advanced' },
+		],
+	},
 ];
 
 const advancedFilters = {
-	status: {
-		labels: {
-			add: 'Order Status',
-			remove: 'Remove order status filter',
-			rule: 'Select an order status filter match',
-			title: 'Order Status',
-		},
-		rules: [
-			{ value: 'is', label: 'Is' },
-			{ value: 'is_not', label: 'Is Not' },
-		],
-		input: {
-			component: 'SelectControl',
-			options: [
-				{ value: 'pending', label: 'Pending' },
-				{ value: 'processing', label: 'Processing' },
-				{ value: 'on-hold', label: 'On Hold' },
-				{ value: 'completed', label: 'Completed' },
-				{ value: 'refunded', label: 'Refunded' },
-				{ value: 'cancelled', label: 'Cancelled' },
-				{ value: 'failed', label: 'Failed' },
+	title: 'Orders Match {{select /}} Filters',
+	filters: {
+		status: {
+			labels: {
+				add: 'Order Status',
+				remove: 'Remove order status filter',
+				rule: 'Select an order status filter match',
+				title: 'Order Status {{rule /}} {{filter /}}',
+				filter: 'Select an order status',
+			},
+			rules: [
+				{
+					value: 'is',
+					label: 'Is',
+				},
+				{
+					value: 'is_not',
+					label: 'Is Not',
+				},
 			],
+			input: {
+				component: 'SelectControl',
+				options: Object.keys( orderStatuses ).map( key => ( {
+					value: key,
+					label: orderStatuses[ key ],
+				} ) ),
+			},
 		},
-	},
-	product_id: {
-		labels: {
-			add: 'Products',
-			placeholder: 'Search products',
-			remove: 'Remove products filter',
-			rule: 'Select a product filter match',
-			title: 'Product',
+		product: {
+			labels: {
+				add: 'Products',
+				placeholder: 'Search products',
+				remove: 'Remove products filter',
+				rule: 'Select a product filter match',
+				title: 'Product {{rule /}} {{filter /}}',
+				filter:'Select products',
+			},
+			rules: [
+				{
+					value: 'includes',
+					label: 'Includes',
+				},
+				{
+					value: 'excludes',
+					label: 'Excludes',
+				},
+			],
+			input: {
+				component: 'Search',
+				type: 'products',
+				getLabels: () => Promise.resolve( [] ),
+			},
 		},
-		rules: [
-			{ value: 'includes', label: 'Includes' },
-			{ value: 'excludes', label: 'Excludes' },
-		],
-		input: {
-			component: 'Search',
-			type: 'products',
-			getLabels: function() {
-				return Promise.resolve( [] );
+		customer: {
+			labels: {
+				add: 'Customer Type',
+				remove: 'Remove customer filter',
+				rule: 'Select a customer filter match',
+				title: 'Customer is {{filter /}}',
+				filter: 'Select a customer type',
+			},
+			input: {
+				component: 'SelectControl',
+				options: [
+					{ value: 'new', label: 'New', },
+					{ value: 'returning', label: 'Returning', },
+				],
+				defaultOption: 'new',
 			},
 		},
 	},

--- a/packages/components/src/table/example.md
+++ b/packages/components/src/table/example.md
@@ -2,7 +2,11 @@
 import { TableCard } from '@woocommerce/components';
 
 const noop = () => {};
-const headers = [ { label: 'Month' }, { label: 'Orders' }, { label: 'Revenue' } ];
+const headers = [
+	{ key: 'month', label: 'Month' },
+	{ key: 'orders', label: 'Orders' },
+	{ key: 'revenue', label: 'Revenue' },
+];
 const rows = [
 	[
 		{ display: 'January', value: 1 },

